gecode/int/element.hh

```diff
@@ -316,6 +316,35 @@ namespace Gecode { namespace Int { namespace Element {
     GECODE_INT_EXPORT virtual ExecStatus propagate(Space& home, const ModEventDelta& med);
   };
 
+ /**
+  * \brief Domain consistent pair-with-offsets propagator
+  *
+  * Requires \code #include <gecode/int/element.hh> \endcode
+  *
+  * \ingroup FuncIntProp
+  */
+ class GECODE_VTABLE_EXPORT PairWithOffsets
```

Comment:
Is there some reason that I'm missing that the code is copied instead of Pair templatized for the view?

Reply:
Pair uses an OffsetView internally, so in a template version we'd need to be able to create an OffsetView of an OffsetView, which seemed more difficult to support than just having two versions of Pair.

Reply:
Ahh, naturally! I missed that when checking.